engine/src/main/scala/cromwell/webservice/WorkflowSubmitBatchActor.scala

```diff
@@ -34,7 +35,9 @@ case class WorkflowManagerBatchIndexedResponse(response: WorkflowManagerResponse
 class WorkflowSubmitIndexedActor(batchSubmitActor: ActorRef, requestHandlerActor: ActorRef,
```

Comment:
At the risk of jumping onto the Thibault bandwagon... `Router`?

OTOH... nah :)